### PR TITLE
API/OP (affine_channel, group_norm, layer_norm, random_crop, unpool, …

### PR DESCRIPTION
**Pull request by lijianshe02**
error message enhancement 
* affine_channel
* group_norm
* layer_norm
* random_crop
* unpool
* log_loss

### CODE REVIEW

File: paddle/fluid/operators/layer_norm_op.cc [hunk 1]
```diff
@@ -27,36 +27,57 @@ class LayerNormOp : public framework::OperatorWithKernel {
...
+              "Input(Scale) is [%d]",
+              ctx->GetInputDim("Scale").size()));
 
       if (ctx->IsRuntime()) {
-        PADDLE_ENFORCE_EQ(ctx->GetInputDim("Scale")[0], right,
-                          "scale should with right");
+        PADDLE_ENFORCE_EQ(
+            ctx->GetInputDim("Scale")[0], right,
+            platform::errors::InvalidArgument(
+                "The first element of Input(Scale)'s dimension should with"
```
Comment (chenwhql) [inline paddle/fluid/operators/layer_norm_op.cc]: should with right?

File: paddle/fluid/operators/layer_norm_op.cc [hunk 2]
```diff
@@ -27,36 +27,57 @@ class LayerNormOp : public framework::OperatorWithKernel {
...
+              "The rank of Input(Bias) must be 1, but received rank of"
+              "Input(Bias) is [%d]",
+              ctx->GetInputDim("Bias").size()));
       if (ctx->IsRuntime()) {
-        PADDLE_ENFORCE_EQ(ctx->GetInputDim("Bias")[0], right,
-                          "bias should with right");
+        PADDLE_ENFORCE_EQ(
+            ctx->GetInputDim("Bias")[0], right,
+            platform::errors::InvalidArgument(
+                "The first element of Input(Scale)'s dimension should with"
```
Comment (chenwhql) [inline paddle/fluid/operators/layer_norm_op.cc]: same above

File: paddle/fluid/operators/layer_norm_op.cc [hunk 3]
```diff
@@ -90,8 +111,11 @@ class LayerNormOpMaker : public framework::OpProtoAndCheckerMaker {
                    "Constant for numerical stability [default 1e-5].")
         .SetDefault(1e-5)
         .AddCustomChecker([](const float &epsilon) {
-          PADDLE_ENFORCE(epsilon >= 0.0f && epsilon <= 0.001f,
-                         "'epsilon' should be between 0.0 and 0.001.");
+          PADDLE_ENFORCE_EQ(epsilon >= 0.0f && epsilon <= 0.001f, true,
+                            platform::errors::InvalidArgument(
+                                "'epsilon' should be between 0.0 and 0.001,"
```
Comment (chenwhql) [inline paddle/fluid/operators/layer_norm_op.cc]: `epsilon's value in LayerNormOp`, if here error occured, can't know it is which op's error

File: paddle/fluid/operators/layer_norm_op.cc [hunk 4]
```diff
@@ -100,7 +124,10 @@ class LayerNormOpMaker : public framework::OpProtoAndCheckerMaker {
         .SetDefault(1)
         .AddCustomChecker([](const int &begin_norm_axis) {
           PADDLE_ENFORCE_GT(begin_norm_axis, 0,
-                            "'begin_norm_axis' should be greater than zero.");
+                            platform::errors::InvalidArgument(
+                                "'begin_norm_axis' should be greater than zero."
```
Comment (chenwhql) [inline paddle/fluid/operators/layer_norm_op.cc]: same above

File: paddle/fluid/operators/log_loss_op.cc [hunk 5]
```diff
@@ -23,25 +23,36 @@ class LogLossOp : public framework::OperatorWithKernel {
...
+              "rank of Input(Labels), but received rank of Input(Predicted)"
+              "is [%s], received rank of Input(Labels) is [%s].",
+              pred_dims, label_dims));
     }
     PADDLE_ENFORCE_EQ(pred_dims.size(), 2,
-                      "The rank of Input(Predicted) must be 2 and the shape is "
-                      "[batch_size, 1].");
+                      platform::errors::InvalidArgument(
+                          "The rank of Input(Predicted) must be 2,"
+                          "But received rank of Input(Predicted)"
```
Comment (chenwhql) [inline paddle/fluid/operators/log_loss_op.cc]: `rank` -> `dimension`? refer https://github.com/PaddlePaddle/Paddle/pull/20099, other places also need be changed

File: paddle/fluid/operators/random_crop_op.cc [hunk 6]
```diff
@@ -36,7 +39,10 @@ class RandomCropOp : public framework::OperatorWithKernel {
         PADDLE_ENFORCE_GE(
             x_dim[x_i], shape[shape_i],
             platform::errors::InvalidArgument(
-                "Size of Input(X) must be larger than Attr(shape)"));
+                "Size of Input(X) must be larger than Attr(shape),"
```
Comment (chenwhql) [inline paddle/fluid/operators/random_crop_op.cc]: why here are size?

File: paddle/fluid/operators/layer_norm_op.cc (hunk 1 above)
Comment (lijianshe02) [reply paddle/fluid/operators/layer_norm_op.cc]: 我修改下描述～

File: paddle/fluid/operators/layer_norm_op.cc (hunk 2 above)
Comment (lijianshe02) [reply paddle/fluid/operators/layer_norm_op.cc]: 好的，谢谢～

File: paddle/fluid/operators/layer_norm_op.cc (hunk 3 above)
Comment (lijianshe02) [reply paddle/fluid/operators/layer_norm_op.cc]: 好的，我修改下

File: paddle/fluid/operators/layer_norm_op.cc (hunk 4 above)
Comment (lijianshe02) [reply paddle/fluid/operators/layer_norm_op.cc]: 好的，我修改下～

File: paddle/fluid/operators/random_crop_op.cc (hunk 6 above)
Comment (lijianshe02) [reply paddle/fluid/operators/random_crop_op.cc]: 好的，我修改下

**Review by chenwhql — APPROVED** [review]
LGTM

**Pull request merged**